docker/.env

```diff
@@ -39,8 +39,11 @@ PYTHONPATH=/app/pythonpath:/app/docker/pythonpath_dev
 REDIS_HOST=redis
 REDIS_PORT=6379
 
-FLASK_ENV=development
-SUPERSET_ENV=development
-SUPERSET_LOAD_EXAMPLES=yes
+FLASK_ENV=production
+SUPERSET_ENV=production
+SUPERSET_LOAD_EXAMPLES=no
 CYPRESS_CONFIG=false
 SUPERSET_PORT=8088
+
+MONITOR_PAT=a.golovnev:7url2422hg5vnotdejetgcccmtfed3w465way55riaigfclx4whq
```

Comment:
@agolovnev-godeltech maybe it makes sense to put placeholder here and edit this file on target machine?